CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+## Release v2.2.0 - 2020/09/01
+
+### Enhancement
+* `object`: Implement S3 api 'deleteBucketPolicy' and related test cases. [#757](https://github.com/chubaofs/chubaofs/pull/757)
```

Comment:
Project function and code test related information does not need to be reflected in Release Notes, because these contents have nothing to do with functionality.